server/Users/Users.js

```diff
@@ -128,8 +133,24 @@ exports.getContactInformation = (req, res) => {
   });
 };
 
-exports.updateFriendById = (req, res) => {
-  res.status(200).json({});
+exports.updateFriendPrivacy = (req, res) => {
```

Comment:
Does this function have to be so specific that it only updates a friends privacy settings? If someone wanted to change a contacts name in the future, could we rewrite this function to handle that as well?